src/supported-documents/index.ts

```diff
@@ -1,14 +1,21 @@
-import supportedDrivingLicences from './supported-docs-driving_licence.json'
-import supportedNationalIDCards from './supported-docs-national_identity_card.json'
-import supportedResidencePermit from './supported-docs-residence_permit.json'
+/* eslint-disable @typescript-eslint/no-var-requires */
+const supportedDrivingLicences = require('./supported-docs-driving_licence.json')
+const supportedNationalIDCards = require('./supported-docs-national_identity_card.json')
+const supportedResidencePermit = require('./supported-docs-residence_permit.json')
```

Comment:
No strong feelings about this, but just curious why this change was needed since an eslint rule also needs to be disabled because of it?

Reply:
By default, TypeScript should properly recognise JSON modules imported by ES6 imports (`import * from 'something.json'`). But I got errors when using it within our project, so just using `require` imports here as a workaround and try to sort it out later